doc/usr/source/9_other/8_interpreter.rst

```diff
@@ -33,7 +33,7 @@ The overall structure is as follows:
     {
       "var1": "24",
       "var2": false,
-      "var3": "0.5"
```

Comment:
Instead of replacing floating point with fractions, why not add a fourth variable?

Reply:
We are showing input values for two steps. I replaced one floating point instance with a fraction instance.